packages/core/src/blocks/Emoji.tsx

```diff
@@ -0,0 +1,26 @@
+import { createInlineContentSpec } from "../schema";
```

Comment:
You should move this file to a new directory called `inlineContent`, and structure it the same way that `blocks` is structured.